### ISSUE TITLE
No such file or directory: 'data/LIGAND_RAW.tsv'

### ISSUE THREAD

**Issue by amiloudij** [XuhanLiu/DrugEx#23]
how to handle that?

**Comment by martin-sicho** [comment]
Hi, unfortunately the file is missing from the repository it seems. However, note that there is now an improved and self-contained version of DrugEx available here: https://github.com/CDDLeiden/DrugEx If you are still looking for this file specifically, I can also provide it.

@XuhanLiu Do you think you could link to the new repository also in the README.md? Perhaps something along the lines: "This project was now upgraded to an installable package with tutorials and test data. You can find it here: https://github.com/CDDLeiden/DrugEx" Many thanks!

**Comment by amiloudij** [comment]
Can you provide it to me. I am trying to re do it fom scratch and provide bew algorithms may be. I am preparing a thesis. Thank you

**Comment by martin-sicho** [comment]
@amiloudij We have versions of some old files [here](https://drive.google.com/file/d/1_j5tXXwDDD4wiX9DaPmp3cZM_icm1bUZ/view). The names are a bit mangled, but you should be able to find the file you are looking for in the `data` folder. You just need to rename it.

**Comment by amiloudij** [comment]
'data/ligand_mf_brics.txt ERROR....no such file or directory

**Comment by amiloudij** [comment]
> @amiloudij We have versions of some old files [here](https://drive.google.com/file/d/1_j5tXXwDDD4wiX9DaPmp3cZM_icm1bUZ/view). The names are a bit mangled, but you should be able to find the file you are looking for in the `data` folder. You just need to rename it.

'data/ligand_mf_brics.txt ERROR....no such file or directory

**Comment by martin-sicho** [comment]
This file should get generated if the processing of `LIGAND_RAW.txt` went fine. Can you check the files you have in `data`? You can follow the code here to check what to look for: https://github.com/XuhanLiu/DrugEx/blob/master/dataset.py

It is possible there is an error in the names of the files here: https://github.com/XuhanLiu/DrugEx/blob/master/train_graph.py So you might need to edit them.

**Comment by amiloudij** [comment]
I dont have the file voc_
atom.txt

**Comment by martin-sicho** [comment]
This one should be fine to use as well: https://github.com/CDDLeiden/DrugEx/blob/master/testing/clitest/data/voc_graph.txt

**Comment by martin-sicho** [comment]
By the way, if you are planning to add new algorithms and do not need to reproduce the experiments from the original paper exactly, I would really recommend the new packaged version again. It is more user friendly and also extensible. If you are willing to contribute your implementation to open source, we would love to help you get started with it as well.

**Comment by amiloudij** [comment]
I see that you are using a file named voc_atom.txt in the code and we dont have it in the repo.

**Comment by martin-sicho** [comment]
> I see that you are using a file named voc_atom.txt in the code and we dont have it in the repo.

You can see the one I mention here:

> This one should be fine to use as well: https://github.com/CDDLeiden/DrugEx/blob/master/testing/clitest/data/voc_graph.txt

Just rename it.

**Comment by amiloudij** [comment]
@martin-sicho  Yes I want to contribute with you. What kind of help can you help me?

**Comment by martin-sicho** [comment]
I you want to use the updated repo, you can get all basic information with the tutorial: https://github.com/CDDLeiden/DrugEx/tree/master/tutorial We are still improving and updating it, but if you want to use the graph-based transformer or the RNN, you should be covered for most applications. The Python API already offers some customization, but if you want to modify the models a very specific way or add your own, let us know on the issue tracker and we can suggest how to approach it. For more info there is a [README](https://github.com/CDDLeiden/DrugEx) and [documentation](https://cddleiden.github.io/DrugEx/docs/).
